packages/plugin-core/src/utils/AutoCompletable.ts

```diff
@@ -0,0 +1,3 @@
+export type AutoCompletable = {
```

Comment:
Can you add some comments on interface-like types? 

i.e. it's not actually clear to me what else should implement `onAutoComplete`